openshift/overrides/pkg/pipelines/resources/tekton/pipeline/dev-console/0.1/pipeline.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: tekton.dev/v1beta1
 kind: Pipeline
 metadata:
-  name: devconsole-function-pipeline
+  name: devconsole-nodejs-function-pipeline
```

Comment:
will the pipeline be specific for nodejs only?  if so perhaps the filename could be `pipeline-nodejs.yaml` so we could have in future a `pipeline-quarkus.yaml` and so on...

But the yaml seems to me generic and could be used in future for other runtimes..